bin/spice/pkg/cli/cmd/dataset.go

```diff
@@ -66,12 +55,13 @@ spice dataset configure
 		accelerateDataset := strings.ToLower(accelerateDatasetString) == "y"
 
 		dataset := api.Dataset{
-			Name:   datasetName,
-			Type:   api.DATASET_TYPE_OVERWRITE,
-			Source: datasetSource,
+			From: datasetLocation,
+			Name: datasetName,
 			Acceleration: &api.Acceleration{
-				Enabled: accelerateDataset,
-				Refresh: time.Hour,
+				Enabled:         accelerateDataset,
+				RefreshInterval: time.Hour,
+				RefreshMode:     api.REFRESH_MODE_FULL,
+				Retention:       30 * time.Minute,
```

Comment:
```suggestion
```

I don't think we should set a retention by default - especially given its a full refresh